### DISCUSSION TITLE
Save Excel file to PDF

### DISCUSSION THREAD

**Discussion by bboshapham** [ClosedXML/ClosedXML#2311] [Ideas]
Hi Team, It would be great if ClosedXML allowed saving Excel file to PDF. I'm looking forward to it.

**Comment by Pete-Fowler** [comment]
I agree this would be a great feature. Does anyone know of a workaround or way to do this that is free? I mainly only see paid libraries like Gembox.Spreadsheet which costs $890. Don't want to use OpenXML since it would require MS Excel on the servers.

**Reply by igitur** [reply]
ClosedXML is a wrapper around OpenXML. Neither of these libraries require Excel to be installed on the server.

**Comment by igitur** [comment]
Generating any visual representation of a spreadsheet, whether HTML, PDF or anything else would require a renderer. Nothing like this exists in ClosedXML and it is definitely currently out of scope.

Of course, as this is open source, if someone submits well written code to enable this, we would strongly consider it for inclusion.
